playbooks/templates/.env.j2

```diff
@@ -120,7 +120,7 @@ DISCORD_MENTION_USER_ID={{ discord_mention_user_id | default('') }}
 # For dev servers: {{ webportal_common_config.s3_backup_path }}_<server hostname>
```

Comment:
With this change these 3 comments
- `# common path for...`
- `# For prod servers...`
- `# For dev servers...`
are not relevant anymore and should be removed.

Reply:
Removed.